thirdparty/runtime_environments.go

```diff
@@ -58,3 +60,56 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// ImageDiffOptions represents the arguments for getImageDiff. AMI is the starting AMI, and AMI2 is the ending AMI.
+type ImageDiffOptions struct {
+	AMI   string
+	AMI2  string
```

Comment:
Maybe they can have names like `AfterAMI` and `BeforeAMI` (since I think the order matters)